timemachine/fe/lambda_schedule.py

```diff
@@ -74,9 +75,34 @@ def construct_pre_optimized_relative_lambda_schedule(n_windows):
     np.array of size n_windows
     """
     lambda_schedule = np.array(
-        [0.0, 0.02, 0.04, 0.06, 0.07, 0.08, 0.11, 0.13, 0.15, 0.17, 0.18, 0.20, 0.25, 0.32, 0.42]
+        [0.0, 0.04, 0.08, 0.12, 0.14, 0.16, 0.22, 0.26, 0.3, 0.34, 0.36, 0.4, 0.5, 0.64, 0.84, 1.0]
```

Comment:
provenance: Is this the previous lambda_schedule * 2 (with 1.0 appended), or was this the result of further optimization?

Reply:
Right, this is the previous lambda schedule * 2 with 1.0 appended.